eng/pipelines/handlers.yml

```diff
@@ -144,11 +144,12 @@ stages:
             - task: xamops.azdevex.provisionator-task.provisionator@1
               displayName: 'Provision VSWindows'
               inputs:
-                provisioning_script: $(provisionator.vs2019)
+                provisioning_script: $(provisionator.vs)
             - script: dotnet tool restore
               displayName: install dotnet tools
             - powershell: |
-                  & dotnet cake --target=VS-WINUI-CI --configuration=${{ BuildCondition }}
+                  $msbuild = "$env:ProgramFiles/Microsoft Visual Studio/2022/Preview/MSBuild/Current/Bin/MSBuild.exe"
```

Comment:
Because VS 2022 is so wildly different for some reason, it requires a reboot:

```
[14:14:09 WRN] Visual Studio installer exited with code 3010, indicating a reboot is needed.
[14:14:09 WRN] Visual Studio installer requested a reboot--this typically means that installation succeeded, but vswhere will *not* find the newly installed VS until reboot. Treating this item as if it was provisioned successfully!
[14:14:09 INF] Finished!
```

So for now I just hardcode the path to VS2022 MSBuild. We can fix later. I also have a PR which makes all this stuff irrelevant since it is using all the new things like .NET 6 to build.